notebooks/ImvizExample.ipynb

```diff
@@ -94,6 +94,23 @@
     "viewer = imviz.default_viewer"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "81249347-a61b-478e-bcfc-586f0457ff4e",
+   "metadata": {},
+   "source": [
+    "This next cell is only enable for debugging. It captures outputs printed to the underlying application `Output` widget."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "d23931dc",
+   "metadata": {},
+   "source": [
+    "# This is for debugging only.\n",
+    "imviz.app._application_handler.output"
+   ]
+  },
```

Comment:
Do we want to keep this in the public example notebook?

Reply:
Maybe until we can find a way to display which data is currently visible. Every time we have Imviz "hack hour" or something, people ask about that feature. At least for now, we can ask them to enable this cell and see the print out.